### PR TITLE
added myself as participants

### PR DESCRIPTION
**Pull request by blank-27**

### CODE REVIEW

File: Data/participants.json [hunk 1]
```diff
@@ -21,12 +21,22 @@
...
     },
+
+     {
+      "name": "Harshdeep",
+      "email": "lcs2019045@iiitl.ac.in",
+      "profilePicture": "https://avatars0.githubusercontent.com/u/55805028?s=460&v=4",
+      "facebookLink": "https://www.facebook.com/profile.php?id=100010115341689",
+      "githubLink": "https://github.com/blank-27"
+    },
+=======
```
Comment (dojutsu-user) [inline Data/participants.json:32]:
```suggestion
```

File: Data/participants.json [hunk 2]
```diff
@@ -21,12 +21,22 @@
       "facebookLink": "https://www.facebook.com/yk1ax",
       "githubLink": "https://github.com/yk1ax"
     },
+
```
Comment (dojutsu-user) [inline Data/participants.json:24]:
```suggestion
```

File: Data/participants.json [hunk 3]
```diff
@@ -21,12 +21,22 @@
...
+    },
+=======
     {
     "name": "Ashutosh Yadav",
     "email":"yadavashu4488@gmail.com",
      "profilePicture":"https://avatars0.githubusercontent.com/u/55954955?s=460&v=4",
      "facebookLink":"https://www.facebook.com/profile.php?id=100025977280583",
      "githubLink":"https://github.com/yadavashu"
     }
+
```
Comment (dojutsu-user) [inline Data/participants.json:40]:
```suggestion
```

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Comment by rishabhshuklax** [comment]
You already have #34  opened. First close that then open any new PR for the same issue.

**Pull request closed without merging**